server/routers/data/state-of-play.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+
+var ukParties = require('uk-political-parties');
+var _ = require('lodash');
+
+var knownParties = [
```

Comment:
Could do, but I'm using this hardcoded array purely to assist with generating my dummy data – probably won't be needed once I switch over to reading real data out of the Loki DB?